package.json

```
@@ -121,7 +115,7 @@
     "@types/use-sync-external-store": "0.0.3",
     "acorn": "8.8.0",
     "blob-polyfill": "7.0.20220408",
-    "bundlesize": "0.18.1",
+    "bytes": "^3.1.2",
```

Nit: all other dependency versions are pinned
```suggestion
    "bytes": "3.1.2",
```